fairscale/experimental/__init__.py

```diff
@@ -9,6 +9,8 @@
 
 from typing import List
 
-from . import nn, optim, wgit
+# Don't import sub-modules to avoid experimental stuff gets imported directly
```

Comment:
Suggesting changing `Don't import sub-modules to avoid experimental stuff` to `Don't import sub-modules as experimental stuff`